### PR TITLE
Fix #343: Updated CHANGELOG for user name

### LINKED ISSUE #343: Updated CHANGELOG for user name along with commit message
Looks like we somehow missed updating user name in the CHANELOG in the last release.

### PR DESCRIPTION
**Pull request by brgnepal**
Fix #343 

Updated the `CHANGELOG` as per PR merged order along with github username.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,26 +2,28 @@
...
-- Issue #270: DOCKER_API_VERSION not set properly on Windows bug
-- Issue #262: Update CI setup configuration
-- Issue #252: Cosmetic bug: vagrant service-manager box ip --script-readable
-- Issue #207: Remove vagrant-service-manager.spec and stop publishing gem as rpm
-- Issue #196: Introduce unit test harness test
-- Issue #93: Implement style guide based test using rubocop test
-- Issue #90: Add install-cli command enhancement focus:devex
-- Issue #20: --force should force redownloading the TLS certificates documentation
+- Fix #331 Updating test expectation after latest ADB release @hferentschik
+- Fix #207: Removes SPEC file from repository and reference from README @navidshaikh
```
Comment (navidshaikh) [inline CHANGELOG.md]: Remove the colon after issue number

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,26 +2,28 @@
...
+- Fix #321 Adding support for HTTPS URLs for download URLs @budhrg
+- Fix #323 Adding note about CLI default download location @navidshaikh
+- Fix #312 Improving test for install-cli of Kubernetes @hferentschik
+- Fix #312 Adding message to install-cli to indicate that Kubernetes is not supported by this command @budhrg
+- Fix #93 Adding Rubocop configuration @hferentschik
+- Fix #90, #196 and #289 Introducing unit tests using minitest @budhrg
+- Fix #90: Add install-cli command @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: Remove the colon after issue number

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,26 +2,28 @@
...
+- Fix #281 Making download of box file a true dependency of the feature task with update checks @hferentschik
+- Fix #20 Adding documentation about the Docker TLS certification creation @hferentschik
+- Fix #291 Reorganized the Available Commands section @preeticp
+- Fix #285: Native methods to detect host platform @budhrg
+- Fix #287: Removed Unnecessary TODO file @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: for line no 17, 18 Remove the colon after issue number

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,26 +2,28 @@
...
+- Fix #288 Adding build status image to README @hferentschik
+- Fix #278: Acceptance tests for --help option for commands like box, env and service-operations @budhrg
+- Fix #275: Fixes typo in vagrant service-manager --help output @navidshaikh
+- Fix #272: README fix for env default behavior @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: Remove the colon after issue number, for line no 20, 21, 22

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,26 +2,28 @@
...
+- Fix #262 Updating CI config @hferentschik
+- Fix #270: Fix DOCKER_API_VERSION properly on Windows @budhrg
+- Fix #268 Explicitly setting v1.8.4 as dev version for Vagrant as HEAD has switched to Ruby 2.2 @hferentschik
+- Fix #252: Added --script-readable option to box ip @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: and here as well, Remove the colon after issue number

**Comment by hferentschik** [comment]
This per commit. Is this indented? A changelog usually list the issues fixed and not list all applied commits. I am fine adding an author and ordering as to in which order things get applied, but this is just a replication of 'git log' which a change log is not about.

**Comment by brgnepal** [comment]
@hferentschik  We have been following that pattern till now. Check [here](https://github.com/projectatomic/vagrant-service-manager/pull/259/files)

**Comment by hferentschik** [comment]
Rebased and merged

**Pull request closed without merging**